systemd/coreos-installer-reboot-loaddev.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Reboot from zVM LOADDEV after CoreOS Installer
+ConditionVirtualization=zvm
```

Comment:
I think it'd be cleaner without this one and relied only on `coreos.inst.secure_ipl`. That way, we don't even have to think through the case where `coreos.inst.secure_ipl` is given but `ConditionVirtualization=zvm` is somehow false.

Reply:
Hmm, actually now i see an issue here:
- zVM require `vmcp ipl loaddev` to restart with Secure IPL
- LPAR has no `vmcp` and regular `reboot` is enough

Looks like it's better to make a shell script and call appropriate `reboot` depending in virtualization. Wdyt?